xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2814,7 +2814,7 @@ platform.index.doc.author=Last Author
 platform.index.doc.title=Title
 platform.index.doc.fullName=Page
 platform.index._actions=Actions
-platform.index.emptyvalue=
+platform.index.emptyvalue= - 
```

Comment:
Unfortunately this is not the solution. Please see the updated description of the jira issue https://jira.xwiki.org/browse/XWIKI-14436, including the last comment I added.